app/main/views/direct_award.py

```diff
@@ -52,6 +52,22 @@ def list_projects():
     if user_id:
         projects = projects.filter(DirectAwardProject.users.any(id=user_id))
 
+    if "having-outcome" in request.args:
+        projects = projects.filter(
+            db.cast(
+                DirectAwardProject.outcome.has(),
+                db.Boolean,
+            ) == convert_to_boolean(request.args.get("having-outcome"))
```

Comment:
I suppose you could use square brackets here seeing as you already tested it's there, whatevs :)

Reply:
Um yes I guess I could have